build.gradle

```diff
@@ -93,6 +93,12 @@ dependencies {
 
     // awaitility
     testImplementation "org.awaitility:awaitility"
+
+    // observability
+    runtimeOnly 'io.micrometer:micrometer-registry-prometheus'
+
+    // Commons math
+    implementation 'commons-math:commons-math:1.2'
```

Comment:
Could we look for a less old library version ? It is something I have in mind for the other repositories as well.
I would prefer to avoid adding this debt here.